src/local/butler/constants.py

```diff
@@ -44,14 +44,12 @@
     ('linux', ('manylinux2010_x86_64', 'manylinux1_x86_64')),
 ])
 
-if sys.version_info.major == 2:
-  ABIS = {'linux': 'cp27mu', 'windows': 'cp27m', 'macos': 'cp27m'}
-elif sys.version_info.major == 3 and sys.version_info.minor == 7:
+if sys.version_info.minor == 7:
```

Comment:
It is important to have "sys.version_info.major == 3" in both branches for correctness and clarity.

Reply:
yes, sure